qiskit/circuit/library/standard_gates/xx_minus_yy.py

```diff
@@ -0,0 +1,171 @@
+    a coherent rotation by some angle between :math:`|00\rangle` and :math:`|11\rangle`.
+
+    **Circuit Symbol:**
+
+    .. parsed-literal::
+
+             ┌───────────────┐
+        q_0: ┤0              ├
+             │  {XX-YY}(θ,β) │
+        q_1: ┤1              ├
+             └───────────────┘
+
+    **Matrix Representation:**
+
+    .. math::
+
+        \newcommand{\th}{\frac{\theta}{2}}
+
+        R_{XX-YY}(\theta, \beta)\ q_0, q_1 =
+          RZ_1(\beta) \cdot exp(-i \frac{\theta}{2} \frac{XX-YY}{2}) \cdot RZ_1(-\beta) =
+            \begin{pmatrix}
+                \cos(\th)             & 0 & 0 & -i\sin(\th)e^{-i\beta}  \\
+                0                     & 1 & 0 & 0  \\
+                0                     & 0 & 1 & 0  \\
+                -i\sin(\th)e^{i\beta} & 0 & 0 & \cos(\th)
+            \end{pmatrix}
+
+    .. note::
+
+        In Qiskit's convention, higher qubit indices are more significant
+        (little endian convention). In the above example we apply the gate
+        on (q_0, q_1) which results in adding the (optional) phase defined
+        by :math:`beta` on q_1. Instead, if we apply it on (q_1, q_0), the
+        phase is added on q_0. If :math:`beta` is set to its default value
+        of :math:`0`, the gate is equivalent in big and little endian.
+
+        .. parsed-literal::
+
+                 ┌───────────────┐
+            q_0: ┤1              ├
+                 │  {XX-YY}(θ,β) │
+            q_1: ┤0              ├
+                 └───────────────┘
+
+        .. math::
+
+            \newcommand{\th}{\frac{\theta}{2}}
+
+            R_{XX-YY}(\theta, \beta)\ q_1, q_0 =
+            RZ_0(\beta) \cdot exp(-i \frac{\theta}{2} \frac{XX-YY}{2}) \cdot RZ_0(-\beta) =
+                \begin{pmatrix}
+                    \cos(\th)             & 0 & 0 & -i\sin(\th)e^{i\beta}  \\
+                    0                     & 1 & 0 & 0  \\
+                    0                     & 0 & 1 & 0  \\
+                    -i\sin(\th)e^{-i\beta} & 0 & 0 & \cos(\th)
+                \end{pmatrix}
+    """
+
+    def __init__(
+        self,
+        theta: ParameterValueType,
+        beta: ParameterValueType = 0,
+        label: Optional[str] = "{XX-YY}",
+    ):
+        """Create new XX-YY gate.
+
+        Args:
+            theta: The rotation angle.
+            beta: The phase angle.
+            label: The label of the gate.
+        """
+        super().__init__("xx_minus_yy", 2, [theta, beta], label=label)
+
+    def _define(self):
+        """
+        gate xx_minus_yy(theta, beta) a, b {
+            rz(-beta) b;
+            rz(-pi/2) a;
+            sx a;
+            rz(pi/2) a;
+            s b;
+            cx a, b;
+            ry(theta/2) a;
+            ry(-theta/2) b;
+            cx a, b;
+            sdg b;
+            rz(-pi/2) a;
+            sxdg a;
+            rz(pi/2) a;
+            rz(beta) b;
+        }
+        """
+        # pylint: disable=cyclic-import
```

Comment:
There isn't any import here, so this should not be necessary I believe, or is it?
```suggestion
```

Reply:
Done.